src/cache/cache_las.c

```diff
@@ -542,6 +542,77 @@ __las_insert_updates_verbose(WT_SESSION_IMPL *session, WT_BTREE *btree, WT_MULTI
         cache->las_verb_gen_write = ckpt_gen_current;
 }
 
+/*
+ * When threads race modifying a record, we can end up with more than the max
+ * number of modifications in an update list.
+ *
+ * Leave space for double the maximum number on the stack as a best attempt to
+ * avoid heap allocation.
+ *
+ * TODO: Look into reusing code from __wt_value_return_upd.
+ */
+#define WT_MODIFY_ARRAY_SIZE (WT_MAX_MODIFY_UPDATE * 2)
```

Comment:
No reason in particular. It wasn't visible in `cache_las.c` and I chose double since that seemed less arbitrary than 10 more.

I can either change them to be the same or I could put the `#define` in `btmem.h` where it's visible to everyone.

Reply:
Both seem pretty arbitrary to me. :)

I guess I have a mild preference for `btmem.h`, next to the `WT_MAX_MODIFY_UPDATE` define.

Reply:
There was a messed-up comment in `btmem.h` I wanted to fix, so I pushed that change in 6ea4acbc6.

Reply:
Thanks!